src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs

```diff
@@ -36,15 +40,19 @@ public async Task AddAsync(PopupPage page)
 
             page.DescendantRemoved += HandleChildRemoved;
 
-            if(UIApplication.SharedApplication.KeyWindow.WindowLevel == UIWindowLevel.Normal)
+            if (UIApplication.SharedApplication.KeyWindow.WindowLevel == UIWindowLevel.Normal)
                 UIApplication.SharedApplication.KeyWindow.WindowLevel = -1;
 
             var renderer = page.GetOrCreateRenderer();
 
-            var window = new PopupWindow
+            var window = new PopupWindow();
+
+            if (IsiOS13OrNewer)
             {
-                BackgroundColor = Color.Transparent.ToUIColor()
-            };
+                _windows.Add(window);
```

Comment:
Why do you keep a window in the list? What reason?

Reply:
@michaelstonis

Reply:
@DevEddy @MichaelRumpler Ok I got it. You try to keep windows in memory while pages are visible. But ios has to do it automatically. It looks like a bug of Xamarin.iOS or iOS.Do you still get this issue even with stable releases of Xamarin.iOS 13 and VS4Mac 8.3?

Reply:
So I just have replicated it. It looks like GC problem. GC removes UIWindow even if it's a kay window. Anyway, this solution looks pretty well. 

Thanks, guys.